apps/api/src/app/message-template/usecases/create-message-template/create-message-template.usecase.ts

```diff
@@ -22,13 +23,17 @@ export class CreateMessageTemplate {
       _creatorId: command.userId,
     });
 
+    item = await this.messageTemplateRepository.findById(item._id);
```

Comment:
I think this is not needed, since the .create method should return the newly created object. Or am I missing something?

Reply:
Like we did not get the same object with create and find, create only seems to return the values that is given to it and find gives you like an empty array for arrays properties.